### PR TITLE
[14.0] sale_order_general_discount_triple: fix onchange

### PR DESCRIPTION
**Pull request by AshishHirapara**
[14.0] sale_order_general_discount_triple: fix onchange

Use `update` instead of `write` within “@onchange” methods to avoid caching issue

### CODE REVIEW

**Review by ivs-cetmix — APPROVED** [review]
Looks good!

**Review by francesco-ooops — APPROVED** [review]
functional review ok

**Comment by AshishHirapara** [comment]
@OCA/crm-sales-marketing-maintainers hey guys, 
I have added myself as a maintainer in this module's manifest, still, I cannot merge this change myself. Can you explain the reason?

**Comment by pedrobaeza** [comment]
You can't grant yourself to be maintainer. That's a security flaw. First it should be approved by a PSC. On following PRs, you will be able to merge.

**Comment by pedrobaeza** [comment]
I see that you are already added in the past, but the case doesn't match exactly (`ashishhirapara` against your real `AshishHirapara`). That can be the problem.

**Comment by francesco-ooops** [comment]
> You can't grant yourself to be maintainer. That's a security flaw. First it should be approved by a PSC. On following PRs, you will be able to merge.

Just to understand correctly: when opening a PR for adding a new module, is it ok for the proposing dev to add themselves as maintainer?

or should that happen in a separate PR?

**Comment by pedrobaeza** [comment]
It's OK. Please read my following comment.

**Comment by pedrobaeza** [comment]
@legalsylvain @sbidoul do you know if the bot is case sensitive on github logins?

**Comment by francesco-ooops** [comment]
@pedrobaeza looks to me like github per se is not: 

https://github.com/ashishhirapara
https://github.com/AshishHirapara

it's the same user

**Comment by francesco-ooops** [comment]
@OCA/crm-sales-marketing-maintainers could this be merged please?

**Comment by francesco-ooops** [comment]
@OCA/crm-sales-marketing-maintainers could this be merged?

**Comment by francesco-ooops** [comment]
@OCA/crm-sales-marketing-maintainers weekly request for merge

**Comment by simahawk** [comment]
@AshishHirapara looks good but please rewrite your commit msg to:

```
sale_order_general_discount_triple: fix onchange

Use `update` instead of `write` to avoid caching issue 
```

Note:
- module name included
- 1 line summary
- explanation in the commit description :wink:

**Comment by AshishHirapara** [comment]
> @AshishHirapara looks good but please rewrite your commit msg to:
> 
> ```
> sale_order_general_discount_triple: fix onchange
> 
> Use `update` instead of `write` to avoid caching issue 
> ```
> 
> Note:
> 
> * module name included
> * 1 line summary
> * explanation in the commit description wink

@simahawk done, thanks for the review.

**Comment by simahawk** [comment]
@AshishHirapara I'm talking about the commit msg, not the PR title. And BTW if you write the commit msg properly since the beginning you get a nice PR title and description automatically :wink:

**Comment by AshishHirapara** [comment]
@simahawk, I made a mistake last time. how's it now?

**Comment by elvise** [comment]
@pedrobaeza looks good for merge, what do you think ?

**Pull request merged**
